public/data/vishal-codes.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Vishal Shinde",
+    "bio": "Hey there I am Vishal, I am deeply passionate about software development & innovations & I also firmly believe in the power of the community. I have arranged various sessions in my college to make students aware of the community and about the advancements in technology. In my free time you can find me with my camera pointing towards nature.",
+    "avatar": "https://github.com/vishal-codes.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/vishal-codes",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/vishaltwts",
+        "icon": "twitter"
+      }
+    ]
+}
```

Comment:
GitHub is highlighting there is an issue here, there needs to are an extra blank line at the end with no spaces

Reply:
ohh okaay, actually I left the blank line but @rohan-kulkarni-25 requested to remove that so I removed it. Thanks for merging😀